### DISCUSSION TITLE
Does gmsh support multi-core parallel？

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#794] [Q&A] [answered]
The problem `Does gmsh support multi-core parallel？` is in this [link](https://gitlab.onelab.info/gmsh/gmsh/-/issues/2269)
I notice you can generate one billion elements,but when I use gmsh genenate 50 millions elements may work slowly, how do you generate one billion elements?
![45d0c99798076403a2b9d5283d78d7a](https://user-images.githubusercontent.com/114474894/218310468-b9c26b46-380f-48d1-b500-a4598ff35afb.png)
By the way,I noticed the LOH1 example in your publication, how do you cut out the incomplete plane, is it in gmsh or in paraview? In gmsh, I found that in Tools-->clipping, only the planar structure can be cut out, and the effect similar to the LOH1 in the figure below cannot be cut out.
![dd93dc18c6eb49d94ad34f82f3a0ad1](https://user-images.githubusercontent.com/114474894/218310352-0ac25146-67d2-4220-bc95-4bb9eb3f0763.jpg)

**Comment by Thomas-Ulrich** [comment, marked as answer]
we use SimModeler for large and complex meshes.
```gmsh -3 -algo hxt $prefix.geo```
Seems a good way to generate large meshes (more quickly) in my experience with gmsh.

**Reply by krenzland** [reply]
That works but at least when I last tried it the hxt meshing algorithm could only deal with simple (eg cube) refinement regions/instructions.

**Reply by buzhiqiuyuebai** [reply]
> That works but at least when I last tried it the hxt meshing algorithm could only deal with simple (eg cube) refinement regions/instructions.

For complex meshes， do you have a better method/algorithm to genetate large meshes?
